backend/src/packages/articles/article.service.ts

```diff
@@ -95,14 +96,14 @@ class ArticleService implements IService {
     return { items: articles.map((article) => article.toObjectWithAuthor()) };
   }
 
-  public async find(id: number): Promise<ArticleBaseResponseDto | null> {
-    const article = await this.articleRepository.find(id);
+  public async find(id: number): Promise<ArticleWithAuthorType | null> {
+    const article = await this.articleRepository.find(+id);
```

Comment:
It take `id` as number and then you cast `id` to number by `+` - is it intended to be so?

Reply:
I hope it was a typo